spec/PhpSpec/Util/ExampleObjectUsingTrait.php

```diff
@@ -0,0 +1,6 @@
+<?php namespace spec\PhpSpec\Util;
```

Comment:
Yes: if the trait is in the same file, there is no issue (due to `ReflectionMethod::getStartLine()` and `ReflectionMethod::getEndLine()` refering to the method defined in the trait). The bug is here only when the trait is in a different file.